app/controllers/clinics_controller.rb

```diff
@@ -5,6 +5,7 @@ class ClinicsController < ApplicationController
 
   def index
     @clinics = Clinic.all
+    render json: @clinics.to_json
```

Comment:
To make it a bit clearer that this endpoint can be either json or HTML, let's throw this in a `respond_to` block. Something like:

```ruby
respond_to do |format|
  format.html
  format.json { render @clinics.to_json }
end
```